ntbs-service/Startup.cs

```diff
@@ -39,12 +40,23 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseSqlServer(Configuration.GetConnectionString("ntbsContext"))
             );
 
+            var auditDbConnectionString = Configuration.GetConnectionString("auditContext");
+
+            services.AddDbContext<AuditDatabaseContext>(options =>
+                options.UseSqlServer(auditDbConnectionString)
```

Comment:
As you suggested - makes sense :) I still needed to pass in the connection string again below, because the context here is just what gets registered and used directly by the application (for manual logging), but the Audit.NET framework needs to get the connection string too

Reply:
Cool, I think the separation makes sense.
Just to make sure I'm clear on it -the only place wehre it gets DIed (atm) is the AuditService, isn't it?

Reply:
Yes exactly, and I don't think we should use it outside of the AuditService!